tests/unit/dataset/dataset_files_test.py

```diff
@@ -187,113 +187,99 @@
 
 
 @pytest.mark.parametrize(
-    'read_kwargs',
+    ('load_kwargs', 'expected_samples'),
     [
         pytest.param(
             {'patterns': PATTERNS, 'schema': {'trial_id': pl.Int64}},
-            id='read_kwargs_dict',
+            EXPECTED_DF_PATTERNS,
+            id='patterns',
         ),
         pytest.param(
             None,
-            id='read_kwargs_none',
+            EXPECTED_DF_NO_PATTERNS,
+            id='none',
         ),
     ],
 )
-@pytest.mark.parametrize(
-    'load_function',
-    [None, 'from_asc'],
```

Comment:
the `None` case is not supported anymore as the `load_function` will now already be filled in the `fileinfo` dataframe.

I don't consider it a breaking change, as the function is private.